build/tooling/vendormanifests.go

```diff
@@ -0,0 +1,10 @@
+// This is used for vendoring frameworks to Gatekeeper.
+
+//go:build tools
+// +build tools
```

Comment:
I think the comment line under //go:build needs to be empty?

Taking this from:

```
Constraints may appear in any kind of source file (not just Go), but they must appear near the top of the file, preceded only by blank lines and other line comments. These rules mean that in Go files a build constraint must appear before the package clause.

To distinguish build constraints from package documentation, a build constraint should be followed by a blank line.
```

in https://pkg.go.dev/cmd/go#hdr-Build_constraints

Reply:
good find. gofmt added the new syntax automatically but didn't leave the empty line.

> The gofmt command will add an equivalent //go:build constraint when encountering the older syntax.

I removed the old syntax

Reply:
Thanks for the fix!